src/libs/actions/Session/index.js

```diff
@@ -97,7 +107,7 @@ function signOutAndRedirectToSignIn() {
         Linking.getInitialURL().then((url) => {
             const reportID = ReportUtils.getReportIDFromLink(url);
             if (reportID) {
-                Report.setLastOpenedPublicRoom(reportID);
+                Onyx.set(ONYXKEYS.LAST_OPENED_PUBLIC_ROOM_ID, reportID);
```

Comment:
Removed Report.js dependency as it created a circular dependency between Report<>Session modules